app/uk/gov/hmrc/debttransformationstub/utils/ifsrulesmasterspreadsheet/impl/IfsRulesMasterData.scala

```diff
@@ -63,19 +63,22 @@ final case class IfsRulesMasterData(
           )
       }
 
-    def useChargeReference(index: Int): Option[Boolean] =
-      Lookup2D.chargeReferenceAt(index).actual match {
-        case "N/A"        => None
-        case "Charge ref" => Some(true)
-        case "ASN"        => Some(false)
-        case "VRN"        => Some(false)
-        case "UTR"        => Some(false)
-        case unknown =>
+    def useChargeReference(index: Int): Option[Boolean] = {
+      val chargeReference: String = Lookup2D.chargeReferenceAt(index).actual
+
+      chargeReference.toLowerCase match {
```

Comment:
There's a slight difference between the "charge ref"s in the Regime Usage for SiA, so I've added some case-insensitive comparison. All data should be preserved in the `val chargeReference`